CONTRIBUTING.md

```diff
@@ -176,15 +176,7 @@ the Code of Conduct.
     is used for all of our Google Docs, particularly proposal drafts.
 -   [Google Calendar](https://calendar.google.com/calendar/embed?src=c_07td7k4qjq0ssb4gdl6bmbnkik%40group.calendar.google.com)
     is used for meeting invites and project reminders. Contributors may add
-    calendar entries for meetings added to discuss details. Standard entries
-    are:
-
-    -   The
-        [weekly sync](https://docs.google.com/document/d/1dwS2sJ8tsN3LwxqmZSv9OvqutYhP71dK9Dmr1IXQFTs/edit?resourcekey=0-NxBWgL9h05yD2GOR3wUisg),
-        where contributors are welcome.
-    -   [Open discussions](https://docs.google.com/document/d/1tEt4iM6vfcY0O0DG0uOEMIbaXcZXlNREc2ChNiEtn_w/edit),
-        which are unstructured meeting slots used for discussing proposals,
-        tooling, and other Carbon topics based on who attends.
+    calendar entries for meetings added to discuss details.
```

Comment:
Thanks for pointing this out! I think it makes sense to remove the links to old docs rather than just updating them (we'll probably forget next time too), but would it make sense to keep the descriptions of the meetings other than that?

Minutes are kept [here](https://drive.google.com/corp/drive/folders/1VssO6kn9-HeKfzPDqBDR0Sy5CUE0OnSs) in the shared drive, with "rolling" marking the current documents, if you'd like to add a link to that.

Reply:
Done.